config/initializers/black_light_silence.rb

```diff
@@ -0,0 +1,8 @@
+# Avalon version five writes a lot of deprecation warnings to
+# stderr (which goes to the apache error log). These warnings
+# are fixed in Avalon version six. 
+if Avalon::VERSION.to_i < 6
+  if [:production, :staging, :test].include?(Rails.env.to_sym)
```

Comment:
Just check if it's not development? Instead of looping this array and checking every other environment other then development?

```ruby
if !Rails.env.development? 
```
or use an `unless` instead of the `if bang`